lib/lte_link_control/lte_lc.c

```diff
@@ -171,7 +157,7 @@ static void at_handler_cereg(const char *response)
 	enum lte_lc_lte_mode lte_mode;
 	struct lte_lc_psm_cfg psm_cfg = {0};
 
-	LOG_DBG("+CEREG notification: %s", response);
+	LOG_DBG("+CEREG notification: %.*s", strlen(response) - strlen("\r\n"), response);
```

Comment:
Nit: can use sizeof -1 for `\r\n` to save a few instructions

Reply:
I tried changing the code to use `sizeof()` instead, but at least the flash size of the binary is exactly the same in both cases. I believe the compiler actually optimizes the `strlen()` call out with a constant string.

Reply:
Ah, then it's all good